### PR TITLE
l10n: uk: update translation.

### PR DESCRIPTION
**Pull request by arkid15r**

### CODE REVIEW

File: po/uk.po [hunk 1]
```diff
@@ -2467,28 +2462,26 @@ msgid ""
 msgstr ""
 
 #: builtin/am.c:2224 builtin/bisect.c:177 worktree.c:237
-#, fuzzy, c-format
+#, c-format
 msgid "failed to read '%s'"
-msgstr "не вдалося виконати stat '%s'"
+msgstr "не вдалося прочитати “%s”"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не вдалося прочитати \"%s\""
```

File: po/uk.po [hunk 2]
```diff
@@ -2467,28 +2462,26 @@ msgid ""
...
 
 #: builtin/am.c:2306
-#, fuzzy, c-format
+#, c-format
 msgid "options '%s=%s' and '%s=%s' cannot be used together"
-msgstr "опції '%s' і '%s' не можна використовувати разом"
+msgstr "опції “%s=%s” and “%s=%s” не можна використовувати разом"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "опції \"%s=%s\" and \"%s=%s\" не можна використовувати разом"
```

File: po/uk.po [hunk 3]
```diff
@@ -2622,9 +2610,8 @@ msgid "key-id"
 msgstr ""
 
 #: builtin/am.c:2436 builtin/rebase.c:1129
-#, fuzzy
 msgid "GPG-sign commits"
-msgstr "нові коміти, "
+msgstr "підписаті коміти GPG-підписом"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "підписати коміти GPG-підписом"
```

File: po/uk.po [hunk 4]
```diff
@@ -2666,14 +2652,12 @@ msgid "interactive mode requires patches on the command line"
 msgstr ""
 
 #: builtin/apply.c:7
-#, fuzzy
 msgid "git apply [<options>] [<patch>...]"
-msgstr "git switch [<опції>] [<гілка>]"
+msgstr "git apply [<опції>] [<латка>…]"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "git apply [<опції>] [<латка>...]"
```

File: po/uk.po [hunk 5]
```diff
@@ -2467,28 +2462,26 @@ msgid ""
...
+#, c-format
 msgid "options '%s=%s' and '%s=%s' cannot be used together"
-msgstr "опції '%s' і '%s' не можна використовувати разом"
+msgstr "опції “%s=%s” and “%s=%s” не можна використовувати разом"
 
 #: builtin/am.c:2337
-#, fuzzy
 msgid "git am [<options>] [(<mbox> | <Maildir>)...]"
-msgstr "git clone [<опції>] [--] <сховище> [<директорія>]"
+msgstr "git am [<опції>] [(<скринька> [<поштова директорія>)…]"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "git am [<опції>] [(<скринька> [<поштова директорія>)...]"
```

File: po/uk.po (hunk 1 above)
Comment (arkid15r) [reply po/uk.po]: 👍

File: po/uk.po (hunk 3 above)
Comment (arkid15r) [reply po/uk.po]: Fixed!

File: po/uk.po (hunk 5 above)
Comment (arkid15r) [reply po/uk.po]: Good catch!

File: po/uk.po [hunk 6]
```diff
@@ -2724,24 +2707,24 @@ msgid "git bisect run <cmd>..."
...
 msgid "cannot open file '%s' in mode '%s'"
-msgstr "неможливо закрити записаний %s файл '%s'"
+msgstr "неможливо відкрити файл \"%s\" у режимі \"%s\""
 
 #: builtin/bisect.c:137
-#, fuzzy, c-format
+#, c-format
 msgid "could not write to file '%s'"
-msgstr "неможливо записати %s файл '%s'"
+msgstr "не вдалося записати в файл \"%\"’"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "не вдалося записати в файл \"%s\""
```

File: po/uk.po [hunk 7]
```diff
@@ -2855,29 +2838,27 @@ msgid ""
...
 #: builtin/bisect.c:598
 #, c-format
 msgid "could not open '%s' for appending"
 msgstr ""
 
 #: builtin/bisect.c:721 builtin/bisect.c:734
-#, fuzzy
 msgid "'' is not a valid term"
-msgstr "'%s' не є дійсним імʼям віддаленого сховища"
+msgstr "‘’ не є допустимим терміном"
```
Comment (kasya) [inline po/uk.po]:
```suggestion
msgstr "\"\" не є допустимим терміном"
```

**Review by kasya — COMMENTED** [review]
👍🏼

**Pull request merged**